Client/Application/TestAppDelegate.swift

```diff
@@ -173,7 +173,11 @@ class TestAppDelegate: AppDelegate {
 
     override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Speed up the animations to 100 times as fast.
-        defer { application.keyWindow?.layer.speed = 100.0 }
+        defer {
+            if let keyWindow = UIWindow.key {
```

Comment:
Using `UIWindow.keyWindow` it wasn't working? [(link)](https://github.com/mozilla-mobile/firefox-ios/blob/31e22aa0d7fe94629ebaba991e707fc917b0e1e3/Client/UIWindowExtensions.swift#L8)

Reply:
yes, indeed! Sorry, I didn't realized that that was already there 😄